### ISSUE TITLE
"//css_dir" should allow directory exclusions

### ISSUE THREAD

**Issue by oleg-shilo** [oleg-shilo/cs-script#93]
Triggered by https://github.com/oleg-shilo/cs-script.npp/issues/5

Since probing dirs do not allow user to control the order of probing it would be beneficial to allow exclusion of some probing directories (e.g. configured globally) when they conflict with the user probing logic.

Th proposed syntax `//css !<dir>`.
```C#
//css_dir %ProgramFiles(x86)%\Microsoft Visual Studio\2017\Enterprise\MSBuild\15.0\Bin
//css_dir !%CSScriptNpp_dir%\Roslyn
using System;
using Microsoft.Build.Framework;
public class Program
{
    static public void Main(string[] args)
    {
	Console.WriteLine(typeof(IBuildEngine).Assembly.Location);
    }
}
```

**Comment by maettu-this** [comment]
Just stumbled across this new item. I suggest to also provide `%%` variables to exclude the implicit probing directories listed in #94 without having to expand the respective folder path.

**Comment by oleg-shilo** [comment]
This Issue is kind of being on hold now (or at least delayed).

It was originally triggered by https://github.com/oleg-shilo/cs-script.npp/issues/5. 
My initial thought was that proving exclusion mechanism would be a low effort work around for the original [problem](https://github.com/oleg-shilo/cs-script.npp/issues/5). 

However when I started working on it I found out that the original problem can be solved directly - #94  and the proposed exclusion mechanism is in fact requiring medium-to-high effort. 

Thus this feature request is still valid though it may come later.
